package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -898,7 +898,7 @@ def run(self, **kwargs):
                                quiet=kwargs.get('quiet', None),
                                default=True)
         pm = ProgressMeter(len(frames),
-                           format="HBonds frame %(step)5d/%(numsteps)d [%(percentage)5.1f%%]\r",
+                           format="HBonds frame %(current_step)5d: %(step)5d/%(numsteps)d [%(percentage)5.1f%%]\r",
```

Comment:
Could you switch this to the new format string 

https://github.com/MDAnalysis/mdanalysis/blob/70049e3c4d33b884861e2477e1a794020a58bced/package/MDAnalysis/lib/log.py#L292